scripts/generate-google3-docs/resource-reference/templates/cloudids_cloudidsendpoint.tmpl

```diff
@@ -27,6 +27,10 @@
 <td>v1.projects.locations.endpoints</td>
 </tr>
 <tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Documentation</td>
+<td><a class="external" href="https://cloud.google.com/intrusion-detection-system/docs/configuring-ids#api">https://cloud.google.com/intrusion-detection-system/docs/configuring-ids#api</a></td>
```

Comment:
Should be like the following because this link is part of the GCP public doc:
```
<td><a href="/intrusion-detection-system/docs/configuring-ids#api">/intrusion-detection-system/docs/configuring-ids#api</a></td>
```

The special format is only needed when it is a link out side of the GCP public doc site.

Reply:
@gemmahou fyi - I think either checking whether the `REST Resource Documentation` section exists or checking whether an internal link is represented properly can be done via a linter if not too expensive.